### ISSUE TITLE
Image Header / Footer - no repeat - not appearing at all

### ISSUE THREAD

**Issue by PhenX** [dompdf/dompdf#180]
_Original author: vedio...@gmail.com (August 16, 2010 15:42:18)_

This will never work: 

``` html
<script type="text/php">
if ( isset($pdf) ) 
{
$footer = $pdf->open_object();

$pdf->image("home/production/www/tmpl/ogo.jpg", 'jpg', 50, 10, 143, 46);

$pdf->close_object();
$pdf->add_object($footer, "all");
}
</script>

This will print the image only once:
<script type="text/php">
if ( isset($pdf) ) 
{
$pdf->image("home/production/www/tmpl/ogo.jpg", 'jpg', 50, 10, 143, 46);
$footer = $pdf->open_object();

$pdf->image("home/production/www/tmpl/ogo.jpg", 'jpg', 50, 10, 143, 46);

$pdf->close_object();
$pdf->add_object($footer, "all");
}
</script>
```

<b>What version of the product are you using? On what operating system?</b>
- @version 0.5.1.htischer.20090507 
  wihth PHP 5.3.2-1ubuntu4.2

<b>Please provide any additional information below.</b>
The image is ok, I can print it outside of the footer. If I put the second script at the end of the page before the &lt;/body&gt;, I will the image in the last page, but not on the others. 
If I put this second script just after the &lt;body&gt;, I will print the image on the first page only, but not on the others.

I have the same problem with the png version of this image.

In short: I add this Image manually to a pdf document, but the header/footer system refuses to do the job with it.

_Original issue: http://code.google.com/p/dompdf/issues/detail?id=174_
